docs/docs/adrs/adr-014-epochs.md

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 15
+title: Epochs
+---
+# ADR 014: Epochs
+
+## Changelog
+* 2024-01-105: Proposed, first draft of ADR.
+
+## Status
+
+Proposed
+
+## Context
+
+In every block that the provider valset changes, a `VSCPacket` must be sent to every consumer and a corresponding `VSCMaturedPacket` sent back.
+Given that the validator powers may change very often on the provider chain (e.g., the Cosmos Hub), this approach results in a large workload for the relayers. 
+Although the validator powers may change very often, these changes are usually small and have an insignificant impact on the chain's security.
+In other words, the valset on the consumers can be slightly outdated without affecting security. 
+As a matter of fact, this already happens due to relaying delays. 
+
+As a solution, this ADR introduces the concept of _epochs_. 
+An epoch consists of multiple blocks. 
+The provider sends `VSCPacket`s once per epoch. 
+A `VSCPacket` contains all the valset changes that occurred throughout the epoch. 
+
+## Decision
+
+The implementation of epochs requires the following changes:
+
+- Add a param that sets the number of blocks in an epoch, i.e., `BlocksPerEpoch`. 
```

Comment:
is there a  max value for `BlocksPerEpoch` which should not be exeeded?

Reply:
Not really. Of course reasonable values are needed to not increase too much the delay of propagating validator updates, see https://github.com/cosmos/interchain-security/pull/1550#discussion_r1444354081.